applications/CompressiblePotentialFlowApplication/python_scripts/compute_lift_jump_process_2d.py

```diff
@@ -0,0 +1,57 @@
+import KratosMultiphysics
+import KratosMultiphysics.CompressiblePotentialFlowApplication as CPFApp
+
+def Factory(settings, Model):
+    if(not isinstance(settings, KratosMultiphysics.Parameters)):
+        raise Exception(
+            "expected input shall be a Parameters object, encapsulating a json string")
+
+    return ComputeLiftJumpProcess2D(Model, settings["Parameters"])
+
+# TODO Implement this process in C++ and make it open mp parallel to save time selecting the wake elements
+class ComputeLiftJumpProcess2D(KratosMultiphysics.Process):
+    def __init__(self, Model, settings):
+        # Call the base Kratos process constructor
+        KratosMultiphysics.Process.__init__(self)
+
+        # Check default settings
+        default_settings = KratosMultiphysics.Parameters(r'''{
+            "model_part_name": "",
+            "velocity_infinity": [1.0,0.0,0],
+            "create_output_file": false
+        }''')
+        settings.ValidateAndAssignDefaults(default_settings)
+
+        body_model_part_name = settings["model_part_name"].GetString()
+        if body_model_part_name == "":
+            err_msg = "Empty model_part_name in DefineWakeProcess2D\n"
+            err_msg += "Please specify the model part that contains the body surface nodes"
+            raise Exception(err_msg)
+        self.body_model_part = Model[body_model_part_name]
+
+        self.velocity_infinity = [0,0,0]
+        self.velocity_infinity[0] = settings["velocity_infinity"][0].GetDouble()
+        self.velocity_infinity[1] = settings["velocity_infinity"][1].GetDouble()
+        self.velocity_infinity[2] = settings["velocity_infinity"][2].GetDouble()
+        self.create_output_file = settings["create_output_file"].GetBool()
+
+    def ExecuteFinalizeSolutionStep(self):
+        # Find the Trailing Edge node
+        max_x_coordinate = -1e30
+        for node in self.body_model_part.Nodes:
+            if(node.X > max_x_coordinate):
```

Comment:
Ok fine! However, I would implement it as i mentioned earlier by checking:

```
for node....
   if node.GetValue(KratosMultiphysics.TRAILING_EDGE):
       self.te=node
```

The reason for doing this instead of what you have is that, in case we change the way the trailing edge is found, we will only have to change it in the define_wake_process.py. If we keep it as it is right now, we would need to change the same in both processes. Basically, I would leave the "job" of finding where the trailing_edge node is to the define_wake_process. This way you can remove the max_x_coordinate in your process.

Do you agree?

Reply:
Yes you are right! I just implemented your proposal and I pushed the changes.